src/configurations/adknowledgeportal/routesConfig.ts

```diff
@@ -452,14 +452,15 @@ const routes: GenericRoute[] = [
     ],
   },
   {
-    to: 'News',
     isNested: false,
-    synapseConfigArray: news,
+    displayName: 'News',
+    to: undefined,
```

Comment:
Question. Do we need to pass "undefined" here otherwise the route won't work?

Reply:
I think technically it will work (because of the effective `el.link ?? el.to` in NavBar), but it makes sense to set this to `undefined` if we're using the link (and not using the react router).